fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraFixity.java

```diff
@@ -92,11 +92,12 @@ public RdfNamespacedStream getDatastreamFixity() {
         if (!(resource() instanceof FedoraBinary)) {
             throw new NotFoundException(resource() + " is not a binary");
         }
+        final FedoraBinary binary = (FedoraBinary) resource();
 
         LOGGER.info("Get fixity for '{}'", externalPath);
         return new RdfNamespacedStream(
                 new DefaultRdfStream(asNode(resource()),
-                    ((FedoraBinary)resource()).getFixity(translator())),
+                    binary.getFixity(translator())),
```

Comment:
Whenever possible, it is helpful to minimize combining stylistic/non-functional changes within a functional pull-request.
I would suggest reverting the changes in this file, as they do not appear to serve any functional purpose... unless I am missing something.